app/routes/admin/users/police/list.js

```diff
@@ -0,0 +1,88 @@
+const { routes, views } = require('../../../../constants/admin')
+const { admin } = require('../../../../auth/permissions')
+const UserListModel = require('../../../../models/admin/users/police/user-list')
+const { getUsers } = require('../../../../api/ddi-index-api/users')
+const { getUser } = require('../../../../auth')
+const { getPoliceForces } = require('../../../../api/ddi-index-api/police-forces')
+const { policeOfficerListQuerySchema } = require('../../../../schema/portal/admin/users')
+const { sort } = require('../../../../constants/api')
+
+const getSortOrder = (key, order) => {
+  if (key === 'indexAccess') {
+    return order !== sort.DESC
+  }
+
+  if (order === sort.DESC) {
+    return sort.DESC
+  }
+
+  return sort.ASC
+}
+
+/**
+ * @param {'email'|'policeForce'|'indexAccess'} key
+ * @param order
+ * @return {GetUserOptions['sort']}
+ */
+const getSortOptions = (key, order) => {
+  if (key === 'email') {
+    return {
+      username: getSortOrder('username', order)
+    }
+  }
+  return {
+    [key]: getSortOrder(key, order)
+  }
+}
+
+const getSortColumns = (column, order) => {
+  return {
+    order: getSortOrder(column, order),
+    column: column ?? 'email'
+  }
+}
+
+module.exports = [
+  {
+    method: 'GET',
+    path: `${routes.policeUserList.get}`,
+    options: {
+      auth: { scope: [admin] },
+      validate: {
+        query: policeOfficerListQuerySchema
+      }
+    },
+    handler: async (request, h) => {
+      const backLink = routes.index
+      const policeForce = request.query.policeForce
```

Comment:
Is it not neater to extract these params like so:
const { policeForce, sortKey, sortOrder } = request.query

Reply:
yeah, didn't think about that.  Good idea